uk.co.compucorp.civicrm.hrcore/gulpfile.js

```diff
@@ -14,7 +14,11 @@ var Promise = require('es6-promise').Promise;
   var files = { config: 'site-config.json', tpl: 'backstop.tpl.json' };
   var configTpl = {
     "url": "http://%{site-host}",
-    "credentials": { "name": "%{user-name}", "pass": "%{user-password}" }
+    "credentials": {
```

Comment:
```js
"credentials": {
  "admin"  : { "name": "%{admin-name}"  , "pass": "%{admin-password}"   },
  "manager": { "name": "%{manager-name}", "pass": "%{manager-password}" },
  "staff"  : { "name": "%{staff-name}"  , "pass": "%{staff-password}"   },
}
```

1. Better "admin" than "default"
2. Leave everything as a placeholder, don't assume names and pwds for roles